app-frontend/src/mocks/store/mockStore.ts

```diff
@@ -0,0 +1,42 @@
+import configureStore from 'redux-mock-store';
+import { UserState } from '../../store/reducers/authorization';
+import { LinksState } from '../../store/reducers/links';
+import { SearchedLinkState } from '../../store/reducers/searchedLink';
+import { SearchedLinksState } from '../../store/reducers/searchedLinks';
+
+interface Link {
```

Comment:
types in mocks shouldn't be instantiated. import types from app code where it is necessary.